frontend/javascripts/admin/organization/pricing_plan_utils.ts

```diff
@@ -41,3 +41,27 @@ export function hasPricingPlanExceededStorage(organization: APIOrganization): bo
 export function isUserAllowedToRequestUpgrades(user: APIUser): boolean {
   return user.isAdmin || user.isOrganizationOwner;
 }
+
+export function isPricingPlanGreaterEqualThan(planA: PricingPlanEnum, planB: PricingPlanEnum) {
+  switch (planA) {
+    case PricingPlanEnum.Power:
+    case PricingPlanEnum.PowerTrial:
+    case PricingPlanEnum.Custom:
+      return true;
+
+    case PricingPlanEnum.Team:
+    case PricingPlanEnum.TeamTrial:
+      switch (planB) {
+        case PricingPlanEnum.Power:
+        case PricingPlanEnum.PowerTrial:
+        case PricingPlanEnum.Custom:
+          return false;
+        default:
+          return true;
+      }
+
+    case PricingPlanEnum.Basic:
+    default:
+      return false;
```

Comment:
on a second thought: I think, it would be easier/less error prone to rewrite the logic to something like this:

```js
const PLAN_TO_RANK: {[key: PricingPlanEnum]: number} = {
  [PricingPlanEnum.Basic]: 0,
  [PricingPlanEnum.Team]: 1
  [PricingPlanEnum.TeamTrial]: 1,
  [PricingPlanEnum.Power]: 2,
  [PricingPlanEnum.PowerTrial]: 2,
  [PricingPlanEnum.Custom]: 2,
}
export function isPricingPlanGreaterEqualThan(planA: PricingPlanEnum, planB: PricingPlanEnum) {
  return PLAN_TO_RANK[planA] >= PLAN_TO_RANK[planB];
}
```

Reply:
Great idea!